.travis.yml

```diff
@@ -53,9 +57,7 @@ env:
     - PANDAS=
     - PILLOW=pillow
     - PYPARSING=pyparsing
-    # pytest-timeout master depends on pytest>=3.6.  Testing with pytest 3.1 is
-    # still supported; this is tested by the first matrix entry.
-    - PYTEST='pytest>=3.6,<3.8'
+    - PYTEST='pytest>=3.6'
```

Comment:
Missed a change here (should now be `3.6.1`), will edit corresponding commit and force push to my branch if CI passes.